.github/workflows/build-viz-state.yml

```diff
@@ -6,6 +6,7 @@ on:
 jobs:
   viz-state:
     runs-on: ubuntu-latest
+    continue-on-error: true
```

Comment:
Let's move this under `- if: ${{ inputs.filter != '' }}` since that's the one failing.
```suggestion
```

Reply:
> - if: ${{ inputs.filter != '' }}

That wouldn't help right now. Since it is failing particularly for `filter: Ripple`.

Reply:
> Let's move this under - if: ${{ inputs.filter != '' }} since that's the one failing.

It fails under any filter, including no filter, when the crawler file does not have some fields, so we can leave it as is.

Reply:
In cases like these I personally think it's better to be explicit about what steps fail rather than implicit, but the end result is the same so this is also fine.